tests/integration_tests/test_pytorch_lightning.py

```diff
@@ -87,3 +89,13 @@ def objective(trial: optuna.trial.Trial) -> float:
     study.optimize(objective, n_trials=1)
     assert study.trials[0].state == optuna.trial.TrialState.COMPLETE
     assert study.trials[0].value == 1.0
+
+
+def test_pytorch_lightning_pruning_callback_monitor_is_invalid() -> None:
+
+    study = optuna.create_study(pruner=DeterministicPruner(True))
+    trial = create_running_trial(study, 1.0)
+    callback = PyTorchLightningPruningCallback(trial, "InvalidMonitor")
+
+    with pytest.warns(UserWarning):
+        callback.on_epoch_end(0, {"loss": 1.0})
```

Comment:
Is `on_epoch_end` the correct callback method? No warning is seemingly emitted, although the implementation of the integration module looks correct.